roles/accprovision/manifests/net-operator-controller-deployment.yaml

```diff
@@ -0,0 +1,460 @@
+                  calico_config:
+                    type: object
+                    properties:
+                      net_config:
+                        type: object
+                        properties:
+                          block_size:
+                            type: integer
+                      bgp_config:
+                        type: object
+                        properties:
+                          bgp_secret:
+                            type: string
+                      bgp_peer_config:
+                        type: object
+                        properties:
+                          as_number:
+                            type: integer
+                          racks:
+                            type: array
+                            items:
+                              type: array
+                              items:
+                               type: string
+                  profile:
+                    type: string
+            type: object
+          status:
+            description: AccProvisionInputStatus defines the successful completion of AccProvisionInput
+            properties:
+              status:
+                type: boolean
+            type: object
+        required:
+        - spec
+        type: object
+---
+apiVersion: v1
+data:
+  spec: "{\n    \"acc_provision_input\": {\n        \"operator_managed_config\": {\n\
+    \            \"enable_updates\": false\n        },\n        \"aci_config\": {\n\
+    \            \"system_id\": \"mykube\",\n            \"apic_hosts\": [\n     \
+    \           \"10.1.1.101\"\n            ],\n            \"aep\": \"kube-cluster\"\
+    ,\n            \"tenant\": {\n                \"name\": \"prj_5cc3c29ec36a49cf942ff84f9e8deb30\"\
+    \n            },\n            \"vrf\": {\n                \"name\": \"mykube-vrf\"\
+    ,\n                \"tenant\": \"common\"\n            },\n            \"l3out\"\
+    : {\n                \"name\": \"mykube_l3out\",\n                \"external_networks\"\
+    : [\n                    \"mykube_extepg\"\n                ]\n            }\n\
+    \        },\n        \"registry\": {\n            \"aci_containers_controller_version\"\
+    : \"kmr2-test\", \n            \"aci_containers_host_version\": \"kmr2-test\"\
+    , \n            \"aci_containers_operator_version\": \"kmr2-test\", \n       \
+    \     \"cnideploy_version\": \"kmr2-test\", \n            \"image_prefix\": \"\
+    quay.io/noirolabs\", \n            \"network_operator_version\": \"test\", \n\
+    \            \"openvswitch_version\": \"kmr2-test\", \n            \"opflex_agent_version\"\
+    : \"kmr2-test\"\n        },\n        \"net_config\": {\n            \"infra_vlan\"\
+    : 3801,\n            \"kubeapi_vlan\": 2032,\n\
+    \            \"extern_dynamic\"\
+    : \"151.9.1.1/24\",\n\
+    \            \"node_subnet\": \"15.11.0.2/27\",\n            \"pod_subnet\": \"\
+    15.128.0.1/27\"\n        }\n    }\n }"
```

Comment:
The above spec should not have any fabric-specific hard-coded values.

Reply:
Putting "dummy" as the value